pages/demo/html/noopener-noreferrer-rel.tsx

```diff
@@ -5,34 +5,35 @@ import Demo, { DemoPreview } from '~/components/demo';
 import Header from '~/components/Header';
```

Comment:
Same as in the previous files.

We should only modify where to replace with `<FormattedText>`.